Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM jrei/systemd-debian:10
```

Comment:
Docker has a concept of "official images", should this be based off of one of those instead?

https://docs.docker.com/docker-hub/official_images/
https://hub.docker.com/_/debian

Reply:
I wondered the same. The fork w/ systemd made sense when Debian had sysv init but that isn't the default in 10 (or 9?). Maybe this fork does other things too?